students/AndrewMiotke/session07/HTML_Renderer/html_render.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+# This is the framework for the base class
+class Element: # Did we need the () in Python3+ ?
```

Comment:
Nope! I believe you'll only have to specify the parameters if you use them (inheritance)

Reply:
👍 cool, that's what I thought but figured I should ask.